service/geopm-service.spec.in

```diff
@@ -74,7 +74,6 @@ BuildRequires: python-rpm-macros
 
 Requires: python%{python3_pkgversion}-geopmdpy = %{version}
 Requires: libgeopmd2 = %{version}
-ExclusiveArch: x86_64
```

Comment:
will need to deal with `cpuid.h` before we can remove this (also true for other spec file.